Lib/ufo2ft/__init__.py

```diff
@@ -12,7 +12,7 @@
 
 def compileOTF(ufo, outlineCompilerClass=OutlineOTFCompiler,
                featureCompilerClass=FeatureOTFCompiler, mtiFeaFiles=None,
-               kernWriter=KernFeatureWriter, markWriter=MarkFeatureWriter,
+               kernWriterClass=KernFeatureWriter, markWriterClass=MarkFeatureWriter,
```

Comment:
Nah. it groups the files together, and when clients write their import statements it shows it's opentype. Just "PostProcessor" can be vague.